packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md

```diff
@@ -152,7 +144,29 @@ While there are no technical restrictions on the values for any of the fields in
 
 Lets you specify additional information to make available in the Okta System Log in connection with the call to your hook. You can use this object as you wish, sending any information that would be useful for debugging purposes. In the System Log, content sent in this object is populated into the `inline_hook.response.processed` event.
 
-## Inline Hook Setup
+## Timeout and retry
+
+When Okta calls your external service, it enforces a default timeout of 3 seconds. Okta will attempt at most one retry. A request is not retried if the customer endpoint returns a 4xx HTTP error code. Any 2xx code is considered successful and not retried. If the external service endpoint responds with a redirect, it is not followed.
+
+### Inline Hooks and concurrent rate limits
+
+The Okta process flow that triggered the Inline Hook remains in progress until the response from your external service is received. For process flows initiated by calls to Okta APIs, slow processing times by your external service can cause open API transactions to accumulate, potentially exceeding [Concurrent Rate Limits](/docs/reference/rate-limits/#concurrent-rate-limits).
+
+### Inline Hook timeout behavior
+
+In the case of an Inline Hook timeout or failure, the Okta process flow behavior either continues or stops based on the Inline Hook type:
+
+| Inline Hook        | Inline Hook Failure Behavior                             |
+|--------------------------------| ---------------------------------------------------------|
+| Token Inline Hook | Okta process flow continues with original token returned. |
+| SAML Assertion Inline Hook | Okta process flow continues with original SAML assertion returned. |
+| Password Import Inline Hook | Okta process flow stops and user can't sign in. The password is not imported. Future attempts to sign-in triggers the Inline Hook again. |
+| User Import Inline Hook | Okta import process continues and user is created. |
+| Registration Inline Hook | Okta process flow stops and registration is denied. The user receives the following UI message: "There was an error creating your account. Please try registering again". |
```

Comment:
I'll add "default UI message". Part of the hook does allow sending a custom message to a user denied registration. But in the timeout case, a default message appears. I like including the message here in case someone searches on it to figure out what's happening.